plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```
@@ -743,9 +743,11 @@ codewhisperer.explorer.enabled=\   Enabled
 codewhisperer.explorer.inline_suggestions=Inline Suggestions
 codewhisperer.explorer.learn=Learn
 codewhisperer.explorer.pause_auto=Pause Auto-Suggestions
+codewhisperer.explorer.pause_auto_scans = Pause Auto Code Scans
 codewhisperer.explorer.paused=\   Paused
 codewhisperer.explorer.reconnect=Reconnect
 codewhisperer.explorer.resume_auto=Resume Auto-Suggestions
+codewhisperer.explorer.resume_auto_scans = Resume Auto Code Scans
```

Let's make these the same as vscode extension

```suggestion
codewhisperer.explorer.pause_auto_scans = Pause Auto-Scans
codewhisperer.explorer.paused=\   Paused
codewhisperer.explorer.reconnect=Reconnect
codewhisperer.explorer.resume_auto=Resume Auto-Suggestions
codewhisperer.explorer.resume_auto_scans = Resume Auto-Scans
```